setup/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM python:3.7-alpine
```

Comment:
We could keep the visual distinction between the signed and the unsigned image if we want to by using a build-argument e.g. 
``` 
FROM python:3.7-alpine
ARG MESSAGE="Hello World of signing! I am a simple image sleeping forever!"
RUN echo "echo ${MESSAGE}; sleep infinity;" >> hello.sh
CMD ["sh", "hello.sh"]
``` 
and then change the message during build by passing it as build-arg somehow like
`DOCKER_CONTENT_TRUST=0 docker build -f Dockerfile -t ${IMAGE_PATH}:signed  --build-arg MESSAGE="I am an unsigned image and won't be allowed to the party :-(" .

Reply:
Good point. I, however, do not think that changing what is in the image is actually critical for testing it after initial setup and would for that reason opt for keeping things as simple as possible, as it is already quite complicated.
What do you think?

Reply:
Fair! Just wanted to propose how we could keep the visibility if we thought it was useful ;-)